config/locales/en.yml

```diff
@@ -330,6 +330,12 @@ en:
       previous_version: "Previous (Version %{version})"
       current_version: "Current (Version %{version})"
       keys_deleted: "Keys deleted:"
+    list_filter:
+      title: Filter by sample names or Id
+      clear: Remove all tags
+      description: 'Paste a list of <kbd>,</kbd> seperated sample names or identifiers'
```

Comment:
Can you change `identifiers` to `persistent unique identifiers (puid)`. or something similar. Just for clarity for the user since samples have ids as well

Reply:
The table header has it as `ID` and I don't know if the user would know or care about the implementation of it as a PUID.

Reply:
I guess you're right...